src/components/Entities/EntityDetailsHeader.tsx

```diff
@@ -3,11 +3,12 @@ import { makeStyles, Theme } from '@material-ui/core/styles';
 import ArrowBack from '@material-ui/icons/ArrowBack';
 import * as classnames from 'classnames';
 import { useCommonStyles } from 'components/common/styles';
-import { Project } from 'models';
+import { Project, ResourceIdentifier } from 'models';
 import { getProjectDomain } from 'models/Project/utils';
 import * as React from 'react';
 import { Link } from 'react-router-dom';
 import { Routes } from 'routes';
+import { launchStrings } from './constants';
```

Comment:
Nit: I have always put constants like this in `SCREAMING_CAMEL` case but not sure what others do

Reply:
Nevermind this is a helper method :)

Reply:
Fair point to make!
I don't have a strong preference, especially in TS where more information is available.
But for this repo, the pattern is mostly to not use a screaming case. So I try to maintain that consistency.